app/javascript/components/Header.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { Component, Fragment }  from "react";
+import { Link, Redirect } from 'react-router-dom';
+
+class Header extends Component {
```

Comment:
you sure header is the best name here?

Reply:
is not it he best name any suggestions? nav-bar maybe